src/window/index.ts

```diff
@@ -195,6 +207,7 @@ export async function openKubernetesErrorMessageWindow(titlePart: string, mainMe
     width:  800,
     height: 494,
     parent: getWindow('preferences') ?? undefined,
+    frame:  true
```

Comment:
```suggestion
    frame:  true,
```
(Trailing comma makes the modification easier)